masterlist.yaml

```diff
@@ -7084,12 +7099,43 @@ plugins:
       # version: 1.58
       - crc: 0xD7423A4E
         util: 'SSEEdit v4.0.0'
+  - name: 'imp_helm_Immersive_Armors.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12405/'
+        name: 'Improved Closedfaced Helmets Patches'
+    req:
+      - 'imp_helm_legend.esp'
+    after:
+      - 'Hothtrooper44_ArmorCompilation.esp'
+    clean:
+      # version: 1
+      - crc: 0x5C4A5C28
+        util: 'SSEEdit v4.0.2'
   - name: 'imp_helm_IDPM.esp'
     url:
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12405/'
         name: 'Improved Closedfaced Helmets Patches'
+    req:
+      - 'imp_helm_legend.esp'
     after:
       - 'imp_helm_USSEP.esp'
+      - 'Improved Dragon Priest Masks.esp'
+    clean:
+      # version: 1
+      - crc: 0x135E09DE
+        util: 'SSEEdit v4.0.2'
+  - name: 'imp_helm_USSEP.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12405/'
+        name: 'Improved Closedfaced Helmets Patches'
```

Comment:
You could combine the _url_ for all 3 patches using a regex.
The Lore Weapon Expansion Addons uses a regex if you need an example.
You'd still need to keep a seperate entry for the other metadata.